.editorconfig

```diff
@@ -0,0 +1,24 @@
+root = true
+
+[*]
```

Comment:
That's the thing that confuses me too about editorconfig. I think `*` is fine for a catch-all, but you should double check if the correct files match with the [CLI tool](https://github.com/editorconfig/editorconfig-core-js).